db/migrate/20180410173704_update_division_with_short_name.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateDivisionWithShortName < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Did you test rolling this back? I anticipate that rails won't know how to reverse this, so it might be better to do separate `up` and `down` methods for this one, where `down` changes all the short name fields to `nil`.